mmv1/products/compute/api.yaml

```diff
@@ -11981,6 +11981,149 @@ objects:
+                      name: 'requestHeadersToAdd'
+                      description: |
+                        Headers to add to a matching request before forwarding the request to the backendService.
+                      item_type: !ruby/object:Api::Type::NestedObject
+                        properties:
+                          - !ruby/object:Api::Type::String
+                            name: 'headerName'
+                            description: 'The name of the header.'
+                          - !ruby/object:Api::Type::String
+                            name: 'headerValue'
+                            description: 'The value of the header to add.'
+                          - !ruby/object:Api::Type::Boolean
+                            name: 'replace'
+                            description: |
+                              If false, headerValue is appended to any values that already exist for the header. If true, headerValue is set for the header, discarding any values that were set for that header.
+                              The default value is false.
+                            default_value: false
+                    - !ruby/object:Api::Type::Array
+                      name: 'responseHeadersToRemove'
+                      item_type: Api::Type::String
+                      description: |
+                        A list of header names for headers that need to be removed from the response before sending the response back to the client.
+                    - !ruby/object:Api::Type::Array
+                      name: 'responseHeadersToAdd'
+                      description: |
+                        Headers to add the response before sending the response back to the client.
+                      item_type: !ruby/object:Api::Type::NestedObject
+                        properties:
+                          - !ruby/object:Api::Type::String
+                            name: 'headerName'
+                            description: 'The name of the header.'
+                          - !ruby/object:Api::Type::String
+                            name: 'headerValue'
+                            description: 'The value of the header to add.'
+                          - !ruby/object:Api::Type::Boolean
+                            name: 'replace'
+                            description: |
+                              If false, headerValue is appended to any values that already exist for the header. If true, headerValue is set for the header, discarding any values that were set for that header.
+                              The default value is false.
+                            default_value: false
+          - !ruby/object:Api::Type::NestedObject
+            name: 'retryPolicy'
+            description : |
+              Specifies the retry policy associated with this route.
+            properties:
+              - !ruby/object:Api::Type::Array
+                name: 'retryConditions'
```

Comment:
Should this be required as well?

Reply:
I had a go creating a regional url map with `retryPolicy` defined but not including `retryConditions` and I didn't get an API error with this:

```
{
 "defaultRouteAction": {
  "retryPolicy": {
   "numRetries": 3,
   "perTryTimeout": {
    "nanos": 500,
    "seconds": "0"
   }
  },
  ...
}
```

I agree it feels like something that should be required if the retryPolicy field is included, though. [The docs](https://cloud.google.com/compute/docs/reference/rest/v1/regionUrlMaps#UrlMap.FIELDS.inlinedField_28) don't describe any requirements or default values, and I don't see default values being returned from the API so seems like it's ok to omit